pilot/pkg/model/authorization.go

```diff
@@ -26,44 +26,49 @@ var (
-	AuthorizationPolicy *authpb.AuthorizationPolicy `json:"authorization_policy"`
+type AuthorizationPolicy struct {
+	Name      string                      `json:"name"`
+	Namespace string                      `json:"namespace"`
+	Spec      *authpb.AuthorizationPolicy `json:"spec"`
 }
 
-// AuthorizationPolicies organizes authorization policies by namespace.
-// TODO(yangminzhu): Rename to avoid confusion from the AuthorizationPolicy CRD.
+// AuthorizationPolicies organizes AuthorizationPolicy by namespace.
 type AuthorizationPolicies struct {
-	// Maps from namespace to the v1beta1 Authorization policies.
-	NamespaceToV1beta1Policies map[string][]AuthorizationPolicyConfig `json:"namespace_to_v1beta1_policies"`
+	// Maps from namespace to the Authorization policies.
+	NamespaceToPolicies map[string][]AuthorizationPolicy `json:"namespace_to_policies"`
 
-	// The name of the root namespace. Policy in the root namespace applies to workloads in all
-	// namespaces. Only used for v1beta1 Authorization policy.
+	// The name of the root namespace. Policy in the root namespace applies to workloads in all namespaces.
 	RootNamespace string `json:"root_namespace"`
 }
 
-// GetAuthorizationPolicies gets the authorization policies in the mesh.
+// GetAuthorizationPolicies returns the AuthorizationPolicies for the given environment.
 func GetAuthorizationPolicies(env *Environment) (*AuthorizationPolicies, error) {
 	policy := &AuthorizationPolicies{
-		NamespaceToV1beta1Policies: map[string][]AuthorizationPolicyConfig{},
-		RootNamespace:              env.Mesh().GetRootNamespace(),
+		NamespaceToPolicies: map[string][]AuthorizationPolicy{},
+		RootNamespace:       env.Mesh().GetRootNamespace(),
 	}
 
 	policies, err := env.List(collections.IstioSecurityV1Beta1Authorizationpolicies.Resource().GroupVersionKind(), NamespaceAll)
 	if err != nil {
 		return nil, err
 	}
 	sortConfigByCreationTime(policies)
-	policy.addAuthorizationPolicies(policies)
+	for _, config := range policies {
+		authzConfig := AuthorizationPolicy{
+			Name:      config.Name,
+			Namespace: config.Namespace,
+			Spec:      config.Spec.(*authpb.AuthorizationPolicy),
+		}
+		policy.NamespaceToPolicies[config.Namespace] =
+			append(policy.NamespaceToPolicies[config.Namespace], authzConfig)
+	}
 
 	return policy, nil
 }
 
-// ListAuthorizationPolicies returns the AuthorizationPolicy for the workload in root namespace and the config namespace.
-// The first one in the returned tuple is the deny policies and the second one is the allow policies.
-func (policy *AuthorizationPolicies) ListAuthorizationPolicies(configNamespace string, workloadLabels labels.Collection) (
-	denyPolicies []AuthorizationPolicyConfig, allowPolicies []AuthorizationPolicyConfig) {
+// ListAuthorizationPolicies returns the deny and allow AuthorizationPolicy for the workload in the given namespace.
+func (policy *AuthorizationPolicies) ListAuthorizationPolicies(namespace string, workload labels.Collection) (
```

Comment:
why rename workloadLabels to workload? old var name seems accurate.

Reply:
to make it shorter, the `labels.Collection` should be enough to tell it's labels.